requirements.txt

```diff
@@ -35,6 +36,9 @@ PyGithub
 pytest==2.9.2
 python-bugzilla>=1.2.0
 python-dateutil
+python-keystoneclient
+python-novaclient<7.0
+python-heatclient
```

Comment:
don't know.. Been working on it for a while now, there may be some rift between the code bases. I will do a rebase when it's ready. Please don't review it now